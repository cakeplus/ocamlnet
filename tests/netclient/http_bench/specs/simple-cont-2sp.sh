. helpers.sh

start_test_server -line 1 -file data/simple-cont-2sp -end
trap "stop_test_server" EXIT
request -get / -run
