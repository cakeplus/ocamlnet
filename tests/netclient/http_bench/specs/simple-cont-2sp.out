33DE1D879D16066A00B5A6A24A75B63E
